sycl/CMakeLists.txt

```diff
@@ -210,6 +210,7 @@ configure_file("${version_header}.in" "${version_header}")
 # Copy SYCL headers
 add_custom_target(sycl-headers ALL
 COMMAND ${CMAKE_COMMAND} -E copy_directory ${sycl_inc_dir}/CL ${SYCL_INCLUDE_BUILD_DIR}/CL
+COMMAND ${CMAKE_COMMAND} -E copy_directory ${sycl_inc_dir}/sycl ${SYCL_INCLUDE_BUILD_DIR}/sycl
```

Comment:
SYCL_INCLUDE_BUILD_DIR is defined as follows. So deploy structure will be <root>/include/sycl/sycl/... which looks wierd.
```	
set(SYCL_INCLUDE_DIR "include/sycl")	set(SYCL_INCLUDE_DIR "include/sycl")
set(SYCL_INCLUDE_BUILD_DIR ${LLVM_BINARY_DIR}/${SYCL_INCLUDE_DIR})	set(SYCL_INCLUDE_BUILD_DIR ${LLVM_BINARY_DIR}/${SYCL_INCLUDE_DIR})
```

Reply:
If I understand correctly the driver sets search path to `./include/sycl` , so to make user code find headers using `<CL/sycl.hpp>` and `<sycl/sycl.hpp>` we need to have the following structure:
`./include/sycl/CL/sycl.hpp` and 
`./include/sycl/sycl/sycl.hpp`.